app/static/404.html

```diff
@@ -11,14 +11,23 @@
       sizes="180x180"
       href="/media/images/icons/apple-touch-icon-180x180.png"
     >
+    <link
+    rel="icon"
+    sizes="16x16"
+    href="/media/images/icons/icon-192.png">
```

Comment:
Add the closing tag to another line
```suggestion
    href="/media/images/icons/icon-192.png"
    >
```

Reply:
See apple touch icon markup